doc/source/doc-style/code/required_make.bat

```diff
@@ -0,0 +1,43 @@
+@ECHO OFF
+
+pushd %~dp0
+
+REM Command file for Sphinx documentation
+
+if "%SPHINXBUILD%" == "" (
+	set SPHINXBUILD=sphinx-build
+)
+set SOURCEDIR=source
+set BUILDDIR=build
+set SPHINXOPTS=-j auto -W --keep-going
+
+if "%1" == "" goto help
+if "%1" == "pdf" goto pdf
+
+%SPHINXBUILD% >NUL 2>NUL
+if errorlevel 9009 (
+	echo.
+	echo.The 'sphinx-build' command was not found. Make sure you have Sphinx
```

Comment:
@jorgepiloto Is it OK for me to edit these "echo" lines?

Reply:
Sure, no problem doing that @PipKat.